tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsUnitTest.java

```diff
@@ -0,0 +1,790 @@
+import static kafka.admin.ReassignPartitionsCommand.alterReplicaLogDirs;
+import static kafka.admin.ReassignPartitionsCommand.brokerLevelFollowerThrottle;
+import static kafka.admin.ReassignPartitionsCommand.brokerLevelLeaderThrottle;
+import static kafka.admin.ReassignPartitionsCommand.brokerLevelLogDirThrottle;
+import static kafka.admin.ReassignPartitionsCommand.calculateFollowerThrottles;
+import static kafka.admin.ReassignPartitionsCommand.calculateLeaderThrottles;
+import static kafka.admin.ReassignPartitionsCommand.calculateMovingBrokers;
+import static kafka.admin.ReassignPartitionsCommand.calculateProposedMoveMap;
+import static kafka.admin.ReassignPartitionsCommand.calculateReassigningBrokers;
+import static kafka.admin.ReassignPartitionsCommand.cancelPartitionReassignments;
+import static kafka.admin.ReassignPartitionsCommand.compareTopicPartitionReplicas;
+import static kafka.admin.ReassignPartitionsCommand.compareTopicPartitions;
+import static kafka.admin.ReassignPartitionsCommand.curReassignmentsToString;
+import static kafka.admin.ReassignPartitionsCommand.currentPartitionReplicaAssignmentToString;
+import static kafka.admin.ReassignPartitionsCommand.executeAssignment;
+import static kafka.admin.ReassignPartitionsCommand.findLogDirMoveStates;
+import static kafka.admin.ReassignPartitionsCommand.findPartitionReassignmentStates;
+import static kafka.admin.ReassignPartitionsCommand.generateAssignment;
+import static kafka.admin.ReassignPartitionsCommand.getBrokerMetadata;
+import static kafka.admin.ReassignPartitionsCommand.getReplicaAssignmentForPartitions;
+import static kafka.admin.ReassignPartitionsCommand.getReplicaAssignmentForTopics;
+import static kafka.admin.ReassignPartitionsCommand.modifyInterBrokerThrottle;
+import static kafka.admin.ReassignPartitionsCommand.modifyLogDirThrottle;
+import static kafka.admin.ReassignPartitionsCommand.modifyTopicThrottles;
+import static kafka.admin.ReassignPartitionsCommand.parseExecuteAssignmentArgs;
+import static kafka.admin.ReassignPartitionsCommand.parseGenerateAssignmentArgs;
+import static kafka.admin.ReassignPartitionsCommand.partitionReassignmentStatesToString;
+import static kafka.admin.ReassignPartitionsCommand.replicaMoveStatesToString;
+import static kafka.admin.ReassignPartitionsCommand.topicLevelFollowerThrottle;
+import static kafka.admin.ReassignPartitionsCommand.topicLevelLeaderThrottle;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+@Timeout(60)
+public class ReassignPartitionsUnitTest {
+    @BeforeEach
+    public void setUp() {
+        Exit.setExitProcedure((statusCode, message) -> {
+            throw new IllegalArgumentException(message);
+        });
+    }
+
+    @AfterEach
+    public void tearDown() {
+        Exit.resetExitProcedure();
+    }
```

Comment:
Could we do that in `@BeforeAll` and `@AfterAll`?

Reply:
Yes. Fixed.